erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -385,7 +398,18 @@ void WebRtcConnection::onTransportData(std::shared_ptr<DataPacket> packet, Trans
   if (getCurrentState() != CONN_READY) {
     return;
   }
-  media_stream_->onTransportData(packet, transport);
+  char* buf = packet->data;
+  RtpHeader *head = reinterpret_cast<RtpHeader*> (buf);
+  RtcpHeader *chead = reinterpret_cast<RtcpHeader*> (buf);
+  uint32_t ssrc = head->getSSRC();
+  if (chead->isRtcp()) {
+    ssrc = chead->getSourceSSRC();
+  }
+  forEachMediaStream([packet, transport, ssrc] (const std::shared_ptr<MediaStream> &media_stream) {
+    if (media_stream->isSourceSSRC(ssrc) || media_stream->isSinkSSRC(ssrc)) {
```

Comment:
This is ok. `isSourceSSRC` would make sure packets from a publisher are properly routed while `isSink` would work for feedback packets from a subscriber.